tests/controls/dialogtest.cpp

```diff
@@ -132,4 +134,36 @@ void ModalDialogsTestCase::CustomDialog()
     CPPUNIT_ASSERT_EQUAL( 42, dlg.m_value );
 }
 
+
+class MyModalDialog : public wxDialog
+{
+public:
+    MyModalDialog() : wxDialog (NULL, wxID_ANY, "Modal Dialog")
+    {
+        m_wasModal = false;
+        Bind( wxEVT_INIT_DIALOG, &MyModalDialog::OnInit, this );
+    }
+
+    void OnInit( wxInitDialogEvent& WXUNUSED(event))
+    {
+        m_wasModal = IsModal();
+        CallAfter( &MyModalDialog::EndModal, wxID_OK );
+    }
+
+    const bool WasModal()
```

Comment:
This should be written as `bool WasModal() const`. There is no need to use top-level const like it's currently done as it's completely ignored (returning `bool` or `const bool` is exactly the same) but, OTOH, the method itself should be const (meaning that it doesn't modify the object).